stdlib/public/core/Indices.swift

```diff
@@ -81,6 +81,23 @@ extension DefaultIndices: Collection {
   public var indices: Indices {
     return self
   }
+  
+  @inlinable
```

Comment:
To maintain ABI while letting these backdeploy to earlier Swift releases, we need to mark these three methods `@_alwaysEmitIntoClient` instead of `@inlinable`:
```suggestion
  @_alwaysEmitIntoClient
```